### ISSUE TITLE
Narrator does not read R output in REPL

### ISSUE THREAD

**Issue by MikhailArkhipov** [Microsoft/RTVS#3174]
Open REPL
1+1 [ENTER] -- Narrator reads it

Observed: Narrator does not read R output `[1] 2`. Does not read it in RGUI either

**Comment by MikhailArkhipov** [comment]
https://github.com/dotnet/interactive-window/issues/37
